File: storage.js
let Web3 = require("web3")

let web3 = new Web3(new Web3.providers.HttpProvider("https://ropsten.infura.io/v3/c05bace974064c25ae612ad7e7afb5aa"))

// address and slot loc
let pin = web3.eth.getStorageAt("0x7e3ed5d28db7be6b572bf96f37ae92f98b491a5f", 0).then(console.log)

File: storage.py
from web3 import Web3

# 读取slot

w3 = Web3(Web3.HTTPProvider('http://123.60.36.208:8545/'))

value = w3.eth.getStorageAt("0x21ac0df70A628cdB042Dde6f4Eb6Cf49bDE00Ff7", 2).hex()

print(value)